R/cal-apply-impl.R

```diff
@@ -0,0 +1,82 @@
+#---------------------------------- >> Interval --------------------------------
+apply_interval_impl <- function(object, .data, multi = FALSE, method = "auto") {
+
+  # Iterates through each group
+  ret <- object$estimates %>%
+    purrr::map(~ {
+      apply_interval_column(
+        .data = .data,
+        est_filter = .x$filter,
+        estimates = .x$estimates
+      )
+    }) %>%
+    purrr::reduce(dplyr::bind_rows)
+
+  if (object$type == "binary") {
+    ret[, object$levels[[2]]] <- 1 - ret[, object$levels[[1]]]
+  } else {
+    ols <- as.character(object$levels)
+    rs <- rowSums(ret[, ols])
+    for(i in seq_along(ols)) {
```

Comment:
Would `apply()` be more efficient here?

Reply:
I just tried using `map()` too, and it gave me fits, because it's giving me type and name inconsistencies between running data.frames and rst sources